src/yb/common/wire_protocol.proto

```diff
@@ -117,6 +117,8 @@ message NodeInstancePB {
   // On a freshly initialized server, the first sequence number
   // should be 0.
   required int64 instance_seqno = 2;
+  // The time point when Tserver/master starts up,it is derived from Env::NowMicros()
```

Comment:
nit: Tserver -> tserver
nit: space after the ,

also, let's name the field `start_time_us`, so the precision is reflected (makes it easier to reason about in code, directly from the field name)